agent/tcs/client/client.go

```diff
@@ -67,11 +73,11 @@ func New(url string, cfg *config.Config, credentialProvider *credentials.Credent
 func (cs *clientServer) Serve() error {
 	seelog.Debug("TCS client starting websocket poll loop")
 	if !cs.IsReady() {
-		return fmt.Errorf("Websocket not ready for connections")
+		return fmt.Errorf("tcs client: websocket not ready for connections")
```

Comment:
Should TCS be capitalized here? We capitalize ACS in most places.

Reply:
We're sticking to the style of error strings beginning with a lower case letter.